src/main/java/uk/gov/hmcts/dts/fact/services/admin/AdminCourtFacilityService.java

```diff
@@ -54,6 +55,11 @@ public List<Facility> updateCourtFacility(final String slug, final List<Facility
         final Court courtEntity = courtRepository.findBySlug(slug)
             .orElseThrow(() -> new NotFoundException(slug));
 
+        courtFacilities.forEach((facility) -> {
```

Comment:
Can we amend the unit test for this test? 

Should be able to assert on the new facilities being returned: 

1. that the description and description cy looks as expected with the p tags 
2. that if you send across a facility with an invalid tag (one that isn't allowed) that the function returns an error, or the expected result